### ISSUE TITLE
Reproducing results in original papers

### ISSUE THREAD

**Issue by ashim95** [QData/TextAttack#122]
Hello,

Thanks for this great codebase.

I have been playing around with this repository and found it to work well out of the box. I have a question regarding reproducibility of results from original papers. I know recipes are the full implementations of the papers.  

Do you have any stats on reproducing results in original papers?

Thanks,

**Comment by jxmorris12** [comment]
Hi @ashim95 -- this is a very good point, and one also brought up by @jind11 -- we need to provide results showing how our results compare to those reported in the papers. We've benchmarked the TextFooler and Alzantot-GA implementations and found results close to those in the papers, but we should add a table. 

I'd also like to add a little disclaimer: we're not necessarily trying to match the results reported in every paper. Some implementations have issues or suboptimal implementation decisions that affect the accuracy. In these cases, we should identify and explain the differences, but do our best to provide an optimal implementation. (In other words, we're generally trying to match the algorithms from the papers, but not necessarily trying to copy the code.)

I'm thinking we add a page like `attack_recipes/README.md` that contains a table of comparisons for each attack recipe. First, we have to update our datasets to have options from the train and test set for each dataset. We have samples from the training sets right now, but many papers use examples from the test sets.

Then we can go recipe-by-recipe, benchmark the results, fix any discrepancies, and add the comparisons to `attack_recipes/README.md`. @ashim95 are you interested in helping us benchmark the recipes?

**Comment by heytitle** [comment]
hi,

Thanks for the nice repo! 

May I also suggest to include a table listing the performance of all available pretrained models?

**Comment by jxmorris12** [comment]
@heytitle that's a really good idea, and much more feasible in the short-term! We'll try and add a page with the performance of each model by the end of this week.

**Comment by monk1337** [comment]
Hi, Any update on this issue?

**Comment by jxmorris12** [comment]
@monk1337 We have a comparison of a selection of attacks in our [paper on Arxiv](https://arxiv.org/pdf/2005.05909.pdf): 

<img width="623" alt="Screen Shot 2020-09-11 at 10 50 40 AM" src="https://user-images.githubusercontent.com/13238952/92940075-a30cad00-f41c-11ea-846c-0bc2bc111e68.png">

**Comment by qiyanjun** [comment]
@heytitle is this page the right one: https://textattack.readthedocs.io/en/latest/3recipes/models.html
